templates/redictcluster/server.conf.erb

```diff
@@ -19,6 +19,7 @@ tls-ca-cert-file /etc/ssl/certs/infra.crt
 <% else %>
 port <%= @redictportnum %>
 <% end %>
+maxmemory-policy <%= @maxmemory_policy %>
```

Comment:
and here:
```
<% if @maxmem %>
maxmem "<%= @maxmem %>"
maxmemory_policy <%= @maxmemory_policy %>
<% end %>
```

Reply:
Don't want to force a `maxmem` limit since that's not necessarily an issue for a particular setup.